samples/benchmarks/coremark/sample.yaml

```diff
@@ -62,10 +68,24 @@ tests:
       - nrf52dk/nrf52832
       - nrf5340dk/nrf5340/cpuapp
       - nrf54l15pdk/nrf54l15/cpuapp
+      - nrf54h20dk/nrf54h20/cpuapp
     integration_platforms:
       - nrf52840dk/nrf52840
       - nrf52dk/nrf52832
       - nrf5340dk/nrf5340/cpuapp
       - nrf54l15pdk/nrf54l15/cpuapp
+      - nrf54h20dk/nrf54h20/cpuapp
     tags: ci_build
     extra_args: EXTRA_CONF_FILE="prj_multiple_threads.conf"
+  sample.benchmark.coremark_ppr_xip:
+    sysbuild: true
+    build_only: true
+    platform_allow:
+      - nrf54h20dk/nrf54h20/cpuapp
+    integration_platforms:
+      - nrf54h20dk/nrf54h20/cpuapp
+    tags: ci_build
+    extra_args:
+      SB_CONFIG_APP_CPUNET_RUN=n
+      SB_CONFIG_APP_CPUPPR_RUN=y
+      coremark_SNIPPET=nordic-ppr-xip
```

Comment:
Would the following modification work in the `sysbuild.cmake`?

```
if(SB_CONFIG_APP_CPUPPR_RUN)
  set(coremark_SNIPPET nordic-ppr-xip)
endif()
```

Reply:
It doesn't work - this is some visibility issue. It only works if the sinppet is set in CMakeLists.txt before `find_pavkage(Zephyr...`